_posts/2023-11-16-virginia-beach-city/virginia-beach-city.Rmd

```diff
@@ -0,0 +1,229 @@
+
+-   [2022 November General Election Virginia Beach City House of Representatives Absentee Voting by Precinct](https://results.elections.virginia.gov/vaelections/2022%20November%20General/Site/Absentee/VIRGINIA_BEACH_CITY/Index.html)
+
+We compile the specific data (Election Day, Early Voting, Mailed Absentee, Provisional, and Post-Election) to create overall vote splits by precinct. 249 write-in ballots, which constituted 0.15% of the total vote in Virginia Beach, are discarded.
+
+### Runs and Simulations
+
+We sample 20,000 districting plans for Virginia Beach City across fifteen independent runs of the [Sequential Monte Carlo algorithm](https://alarm-redist.org/redist/reference/redist_smc.html) according to [the relevant criteria](#redistricting-requirements). A [summary of simulations](#summary-of-simulations) can be found below.
+
+### Redistricting Requirements
+
+Per [Virginia and Federal law](redistrictingguidelines.pdf) under [Va. Code Ann. § 24.2-304.04](https://casetext.com/statute/code-of-virginia/title-242-elections/chapter-3-election-districts-precincts-and-polling-places/article-2-congressional-senatorial-and-house-of-delegates-districts/section-242-30404-standards-and-criteria-for-congressional-and-state-legislative-districts) redistricting in Virginia Beach must:
+
+1.  have equal populations
+
+2.  provide opportunities for racial/ethnic representation
+
+3.  preserve communities of interest (Va. Code Ann. § 24.2-304.04(5).)
+
+4.  be contiguous (Va. Code Ann. § 24.2-304.04(6).)
+
+5.  be compact (Va. Code Ann. § 24.2-304.04(7).)
+
+In the [2021 Virginia Redistricting](redistrictingguidelines.pdf), the following requirements were considered in order of priority:
+
+1.  be contiguous, compact, and equal in population
+
+2.  preserve existing political subdivisions
+
+3.  create clearly discernible boundaries
+
+4.  be reasonably compact
+
+5.  maintain neighborhoods who share similar social, cultural, and economic interests
+
+### Constraints
+
+Three [constraints](https://alarm-redist.org/redist/reference/constraints.html) are added to simulations to promote a majority black district in simulated plans:
+
+1.  A constraint is created that strongly promotes a plan where one district has a black voting age population of at least 55%.
+
+2.  A constraint is created that discourages where the proportion of the black voting age population is less than 40%.
+
+3.  A constraint is created that encourages districts where the black voting age population is less than 20%. Combined, these three constraints promotes districts that will either have a cohesive black voting age population above 55% *or* a small black voting age population. This keeps historically black communities together by discouraging districts that have a percentage of black voting-age population between 20-40%.
+
+### Neighborhoods
```

Comment:
Similar to what we discussed in prior meetings, tell us what neighborhoods you decided to go with at the beginning of this section, and then go into your process of elimination for other types of neighborhood boundaries.

Reply:
Definitely makes sense- moved it up!